pingen/controllers/main.py

```diff
@@ -16,10 +17,13 @@
 
 class PingenController(http.Controller):
     def _verify_signature(self, request_content):
+        if config["test_enable"]:
```

Comment:
This should be tested. Pls drop this change and mock the method in tests when needed.

Reply:
if i would remember why did we add it? :thinking: